test.sh

```diff
@@ -26,20 +26,20 @@ python --version
 
 # Install JAX.
```

Comment:
This is also no longer the case, the same thing for line 22. Can you please update the comments to reflect the changed introduced by this PR?